Dockerfile

```diff
@@ -67,3 +67,9 @@ RUN mkdir /opt/rwd/ && \
     | tar -xzC /opt/rwd --strip-components=1 && \
     cd /opt/rwd && \
     pip install --no-cache-dir -r requirements.txt
+
+RUN pip install flask==0.10.1
```

Comment:
I was thinking you should be able to install rwd without having to install flask and the api. Maybe we should make a separate requirements.txt  in src/api